org.gc.dusty.plist

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+  <key>EnvironmentVariables</key>
+  <dict>
+    <key>HOME</key>
+    <string>/var/root</string>
+  </dict>
+  <key>KeepAlive</key>
+  <true/>
+  <key>Label</key>
+  <string>org.gc.dustyd</string>
+  <key>ProgramArguments</key>
+  <array>
+    <string>bash</string>
+    <string>-c</string>
+    <string>export PATH=$PATH:/usr/local/bin; env; dustyd</string>
```

Comment:
oops that's debugging.  The path mod is needed because OSX daemons/agents seem to use a really bare-bones environment, which doesn't have `PATH` set normally